test/accum_swmr_reader.c

```diff
@@ -57,6 +57,7 @@ main(void)
      * by the environment variable.
      */
     driver = HDgetenv(HDF5_DRIVER);
+
```

Comment:
Can you remove the blank line changes? Then we'll at least get this merged in while we discuss #2900.